packages/edit-post/src/components/header/more-menu/index.js

```diff
@@ -19,6 +20,7 @@ import ToolsMoreMenuGroup from '../tools-more-menu-group';
 import WritingMenu from '../writing-menu';
 
 const MoreMenu = ( { showIconLabels } ) => {
+	const anchorRef = useRef();
```

Comment:
Must be missing something. Where does this ref get attached?

Reply:
I was wondering if this one is working specifically because it isn't attached to anything, so it overrides the Popover's default positioning?

Reply:
Yeah this is unusual, I would also expect this ref to be attached to an element